extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -66,16 +73,16 @@ export class ResourceTypePickerDialog extends DialogBase {
 			return true;
 		});
 		tab.registerContent((view: azdata.ModelView) => {
-			const tableWidth = 1126;
```

Comment:
hmm, why are we reducing the table size?

Reply:
I will remove this change